packages/react-scripts/modernizr-config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "minify": true,
+    "feature-detects": [
+      "test/customevent",
+      "test/dom/intersection-observer",
+      "test/es6/promises",
+      "test/network/fetch",
+      "test/url/parser"
```

Comment:
Looked at `packages/react-scripts/polyfills.js` and usage of these across apps to gather this list.